### PR TITLE
highlighting new players function

### PR DESCRIPTION
**Pull request by MustaKotka**
adding a new function to support highlighting new players in playeroverview

### CODE REVIEW

File: src/logic/utils.ts [hunk 1]
```diff
@@ -36,3 +36,12 @@ export function getAverageWinTurn(player: Player) {
     // Return as string rounded to nearest tenth
     return average.toFixed(1);
 }
+
+export function isNewlyQualifiedPlayer(player: Player) {
+    const dateOffset = new Date();
+    dateOffset.setDate(dateOffset.getDate() - NEW_PLAYER_HIGHLIGHT_DAYS)
```
Comment (davidpchi) [inline src/logic/utils.ts:42]: end lines with semicolon

File: src/logic/utils.ts [hunk 2]
```diff
@@ -36,3 +36,12 @@ export function getAverageWinTurn(player: Player) {
...
+    const lastIndex = player.matches.length - 1
```
Comment (davidpchi) [inline src/logic/utils.ts:43]: end lines with semicolon

File: src/logic/utils.ts [hunk 3]
```diff
@@ -36,3 +36,12 @@ export function getAverageWinTurn(player: Player) {
...
+    if (PLAYER_MINIMUM_GAMES_REQUIRED <= player.matches.length && player.matches.length <= 15 && dateOffset < player.matches[lastIndex].date) {
```
Comment (davidpchi) [inline src/logic/utils.ts:44]: return false otherwise, right now there is no return if this statement is false.

File: src/logic/utils.ts [hunk 4]
```diff
@@ -36,3 +36,12 @@ export function getAverageWinTurn(player: Player) {
     // Return as string rounded to nearest tenth
     return average.toFixed(1);
 }
+
+export function isNewlyQualifiedPlayer(player: Player) {
```
Comment (davidpchi) [inline src/logic/utils.ts:40]: add a description of this method using: "/** "

**Review by davidpchi — CHANGES_REQUESTED** [review]

File: src/logic/utils.ts (hunk 1 above)
Comment (MustaKotka) [reply src/logic/utils.ts]: ended line with semicolon.

**Review by davidpchi — APPROVED** [review]

**Pull request merged**